commands/build.go

```diff
@@ -501,7 +501,7 @@ func buildCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 	flags.Var(options.ulimits, "ulimit", "Ulimit options")
 
 	if isExperimental() {
-		flags.StringVar(&options.invoke, "invoke", "", "Invoke a command after the build. BUILDX_EXPERIMENTAL=1 is required.")
+		flags.StringVar(&options.invoke, "invoke", "", "Invoke a command after the build [experimental]")
```

Comment:
In the CLI with `docker buildx build --help` users will not know how to enable it without looking at docs. Maybe we should also be consistent for the CLI with:

```suggestion
		flags.StringVar(&options.invoke, "invoke", "", "Invoke a command after the build. BUILDX_EXPERIMENTAL=1 env var required.")
```

Reply:
I've added a little note in the docs page to instruct users how to enable the `[experimental]` flags - which I think should maybe be enough?

The arguments shouldn't appear in a normal `--help`, unless `BUILDX_EXPERIMENTAL` is already set, so I'm not sure we need to mention it in the `--help`, it feels like something to call out in docs?

Reply:
> The arguments shouldn't appear in a normal `--help`, unless `BUILDX_EXPERIMENTAL` is already set,

Ah good point :+1: